### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by silviyaboteva**
Part of: https://github.com/telerik/katana/issues/1231

@nrch can you share your feedback on the Nouvelle overview description?

### CODE REVIEW

File: packages/classic/README.md [hunk 1]
```diff
@@ -119,11 +117,9 @@ To ensure version predictability, we use [Semantic Versioning](https://semver.or
 
 This package is part of the following suites:
 
-* [Kendo UI for Angular](http://www.telerik.com/kendo-angular-ui/)
-* [KendoReact](http://www.telerik.com/kendo-react-ui/)
-* [Kendo UI for jQuery](http://www.telerik.com/kendo-ui)
-* [UI for ASP.NET MVC](http://www.telerik.com/aspnet-mvc)
-* [UI for ASP.NET Core](http://www.telerik.com/aspnet-core-ui)
+* [Kendo UI for jQuery](https://www.telerik.com/kendo-ui)
+* [UI for ASP.NET MVC](https://www.telerik.com/aspnet-mvc)
```
Comment (nrch) [inline packages/classic/README.md:121]: The product names include "Telerik" - please change to
Telerik UI for ASP.NET MVC (same for Core)

File: packages/classic/README.md [hunk 2]
```diff
@@ -119,11 +117,9 @@ To ensure version predictability, we use [Semantic Versioning](https://semver.or
...
+* [UI for ASP.NET Core](https://www.telerik.com/aspnet-core-ui)
 
 All available Kendo UI commercial licenses may be obtained at http://www.telerik.com/purchase/kendo-ui.
```
Comment (nrch) [inline packages/classic/README.md:124]: Visit the [Kendo UI Pricing page](https://www.telerik.com/purchase/kendo-ui) to review Kendo UI commercial licensing and pricing.

File: packages/nouvelle/docs/index.md [hunk 3]
```diff
@@ -7,7 +7,9 @@ position: 1
 
 # Nouvelle Theme Overview
 
-> Kendo UI Nouvelle Theme is currently in development stage and it is, currently, covering the base components. Please vote for it if you want a full coverage: https://feedback.telerik.com/themes/1432136-add-theme-based-on-css-variables.
+> Kendo UI Nouvelle Theme is currently experimental and covers the base components such as buttons, inputs, etc. We have an official feature request that you can vote for if you want a full coverage: https://feedback.telerik.com/themes/1432136-add-theme-based-on-css-variables. Once scoped and planned, we will continue with the implementation. The updates will be posted in [this public issue](https://github.com/telerik/kendo-themes/issues/798).
```
Comment (nrch) [inline packages/nouvelle/docs/index.md:10]: The Kendo UI Nouvelle Theme is currently experimental and covers base components such as buttons, inputs, etc. You can vote for it in this [Feedback Portal feature request](https://feedback.telerik.com/themes/1432136-add-theme-based-on-css-variables) if you want full coverage. Any updates will be posted in [this public issue](https://github.com/telerik/kendo-themes/issues/798).

Comment (nrch) [reply packages/nouvelle/docs/index.md:10]: This sounds like the general way things go, but doesn't give any information, so I cut it: "Once scoped and planned, we will continue with the implementation."
If we want to give information about the stage of execution we are at, perhaps we can say "The full implementation of this theme is yet to be scoped and planned."

File: packages/nouvelle/docs/index.md [hunk 4]
```diff
@@ -7,7 +7,9 @@ position: 1
...
+
+In is root, the Kendo UI Nouvelle Theme is a CSS variables based for the Kendo UI components. [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) are properties you set in your stylesheets that reuse throughout the application.
```
Comment (nrch) [inline packages/nouvelle/docs/index.md:12]: In its essence, the Kendo UI Nouvelle Theme is a theme based on CSS variables that can be used with the Kendo UI component libraries. [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) are properties you set in your stylesheets that cab be reused throughout the application.

Comment (nrch) [reply packages/nouvelle/docs/index.md:12]: Could you add a note/phrase about what the other Kendo UI themes are based on so people can better understand what's different abut this theme?

**Review by nrch — CHANGES_REQUESTED** [review]

**Review by nrch — CHANGES_REQUESTED** [review]
A few improvements suggested

File: packages/classic/README.md [hunk 5]
```diff
@@ -119,13 +117,11 @@ To ensure version predictability, we use [Semantic Versioning](https://semver.or
 
 This package is part of the following suites:
 
-* [Kendo UI for Angular](http://www.telerik.com/kendo-angular-ui/)
-* [KendoReact](http://www.telerik.com/kendo-react-ui/)
-* [Kendo UI for jQuery](http://www.telerik.com/kendo-ui)
-* [UI for ASP.NET MVC](http://www.telerik.com/aspnet-mvc)
-* [UI for ASP.NET Core](http://www.telerik.com/aspnet-core-ui)
+* [Telerik Kendo UI for jQuery](https://www.telerik.com/kendo-ui)
```
Comment (nrch) [inline packages/classic/README.md:120]: We don't need the "Telerik" name in front of Kendo UI.
The full name of all of our brands is "Progress [brand]"
From then on, all the JavaScript UI libraries are Kendo UI and all the .NET - Telerik UI. That is, we don't use "Telerik" in the context of JavaScript.
If in doubt, you can always check our official brand names here: https://www.progress.com/legal/trademarks
We don't need to use the (tm) and (r) unless it's an external property and this is the first product mention.
Same with the long name - we don't need to use "Progress Kendo UI" on our website, just "Kendo UI for jQuery"

File: packages/nouvelle/docs/index.md [hunk 6]
```diff
@@ -7,7 +7,11 @@ position: 1
 
 # Nouvelle Theme Overview
 
-> Kendo UI Nouvelle Theme is currently in development stage and it is, currently, covering the base components. Please vote for it if you want a full coverage: https://feedback.telerik.com/themes/1432136-add-theme-based-on-css-variables.
+> The Kendo UI Nouvelle Theme is currently experimental and covers base components such as buttons, inputs, etc. You can vote for it in this [Feedback Portal feature request](https://feedback.telerik.com/themes/1432136-add-theme-based-on-css-variables) if you want full coverage. Any updates will be posted in [this public issue](https://github.com/telerik/kendo-themes/issues/798).
+
+In its essence, the Kendo UI Nouvelle Theme is a theme based on CSS variables that can be used with the Kendo UI component libraries. [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) are properties you set in your stylesheets that can be reused throughout the application.
+
+Unlike the Kendo UI Nouvelle Theme, the other three themes (Default, Bootstrap and Material) are based on [SCSS variables](https://sass-lang.com/documentation/variables).
```
Comment (nrch) [inline packages/nouvelle/docs/index.md:14]: should this be SCSS or SASS, as it is in the URL?

**Review by nrch — APPROVED** [review]
Two small things and we're done

File: packages/classic/README.md (hunk 5 above)
Comment (silviyaboteva) [reply packages/classic/README.md:120]: I understand. Thank for the clarification :)

**Review by joneff — APPROVED** [review]

**Pull request merged**